projects/core/.release-it.changelog.json

```diff
@@ -0,0 +1,22 @@
+{
+  "git": {
+    "requireCleanWorkingDir": true,
+    "requireUpstream": false,
+    "tagName": "core-${version}",
+    "commitMessage": "Release core-${version}",
```

Comment:
We use 'Bumping version to core-${version}' as a commit message

Reply:
Changed to `Bumping core to version ${version}` according to previous tags and commits